go/cmd/vtctldclient/command/schema.go

```diff
@@ -286,8 +285,8 @@ func commandReloadSchemaShard(cmd *cobra.Command, args []string) error {
 }
 
 func init() {
+	ApplySchema.Flags().MarkDeprecated("--allow-long-unavailability", "Deprecated. Large schema migrations should use Online DDL")
```

Comment:
You should skip the deprecated note in the description as pFlag adds that: https://pkg.go.dev/github.com/spf13/pflag#readme-deprecating-a-flag-or-its-shorthand

Reply:
done